logic/placements.py

```diff
@@ -169,5 +182,5 @@ def norm_values(norm: Callable[[str], EIN]):
 )
 
 
-VANILLA_RUPEES = norm_vanilla(RUPEE_CHECKS)
-VANILLA_QUICK_BEETLE_RUPEES = norm_vanilla(QUICK_BEETLE_CHECKS)
+VANILLA_RUPEES = norm_restrict_vanilla(RUPEE_CHECKS)
+VANILLA_QUICK_BEETLE_RUPEES = norm_restrict_vanilla(QUICK_BEETLE_CHECKS)
```

Comment:
```suggestion
VANILLA_RUPEES = norm_force_vanilla(RUPEE_CHECKS)
VANILLA_QUICK_BEETLE_RUPEES = norm_force_vanilla(QUICK_BEETLE_CHECKS)
```
Same as for beedle's shop: Rupees are not patched at all when rupeesanity is set to vanilla